cli/package.json

```diff
@@ -27,6 +27,8 @@
     "gamedev",
     "go",
     "golang",
+    "grain",
+    "grainlang",
```

Comment:
Some other languages are missing from this tag list, but I didn't add others in this PR.